src/VisualStudio/CSharp/Impl/EventHookup/EventHookupSessionManager.cs

```diff
@@ -75,10 +106,11 @@ private void TextBuffer_Changed(object sender, TextContentChangedEventArgs e)
 
             foreach (var change in e.Changes)
             {
-                if (change.OldText.Length > 0 || change.NewText.Any(c => c != ' '))
+                if ((change.OldText.Length > 0 && change.OldText.Any(c => c != ' '))
```

Comment:
don't need the .Length > 0 check.  also, this code really needs commenting to explain what it's trying to accomplish.

Reply:
reverted my change as backspace should dismiss the tooltip